### ISSUE TITLE
Crashed the cluster for few minutes after adding third server

### ISSUE THREAD

**Issue by tamireran** [noobaa/noobaa-core#4437]
### Environment info
- Version: **2.1.2-52e964c**
- Deployment: **AZURE**
- Customer: **demo 2.1.2 clean**

http://40.65.108.21:8080/fe/systems/clean/management/version

**Primary**
IP: 40.65.108.21
93874eab

**Secondary**

IP:52.158.232.114
094e0e23

**Third**
IP: 52.158.236.59
5994c332

### Actual behavior
1. Added another server to the cluster (after #4436). Got the standard message that it will take few minutes. After a minute got an odd error message

![image](https://user-images.githubusercontent.com/3898274/35948211-5a150c96-0c20-11e8-832c-66075cc42365.png)

An attempt to refresh resulted in error 500

![image](https://user-images.githubusercontent.com/3898274/35948303-cee50f76-0c20-11e8-9fe5-65d615d483bb.png)

2018-02-07T16:04:09.584838-08:00 noobaa WebServer[69342]: [L0]  core.server.web_server:: /version caught RPC ERROR INTERNAL mongo_client not connected (collection objectmds) on read system, returning 503

### Expected behavior
1. Replace the message about namespace, I'm not sure how is it relate to anything. Maybe wrong copy paste of a message. 
2. It looks like we restarted the process, and since it's expected, we should show progress on the FE. 

### Screenshots or Logs or other output that would be helpful
[log](http://swiggity2.westus2.cloudapp.azure.com:8080/fe/systems/demo/buckets/customers/objects/clean_cluster_diagnostics_issue_4437.tgz)

**Comment by nimrod-becker** [comment]
No longer relevant due to OVA deployment support removal
